lib/bin/lwm2m_carrier/lwm2m_carrier.rst

```diff
@@ -216,6 +219,14 @@ Below are some of the requirements and limitations of the application while runn
    * The LwM2M carrier library is currently only certified for the *LTE-M* LTE mode.
    * The :option:`CONFIG_LTE_NETWORK_USE_FALLBACK` should be disabled in your application, as seen in the :ref:`lwm2m_carrier` sample project configuration (:file:`nrf/samples/nrf9160/lwm2m_carrier/prj.conf`).
 
+* The LwM2M carrier library registers to receive several AT event reports using the :ref:`at_cmd_readme` and :ref:`at_notif_readme` libraries. The following notifications should not be deregistered by the application:
+
+   * SMS events (AT+CNMI)
+   * Packet Domain events (AT+CGEREP)
+   * EPS Session Management events (AT+CNEC)
+   * OMA-DM IMEI Sync (AT+ODIS)
+   * Network Registration Status events (AT+CEREG)
```

Comment:
It's ok for the application to increase the `+CEREG` level, but not decrease it.